### PR TITLE
Added an alternative script for creating manifests, plus images BT-306

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: scripts/reference_disks/create_images.sh [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    local cloud_object
+
+    cloud_object="$1"
+
+    # URL encode the object using jq. Easier than trying to figure out which python2/python3 is available.
+    # And we're using jq again just below to retrieve the crc32c from the output.
+    # https://stackoverflow.com/questions/37309551/how-to-urlencode-data-into-a-url-with-bash-or-curl#comments-37309630
+    cloud_object_encoded="$(jq --raw-input --raw-output @uri <<<"$cloud_object")"
+
+    # No need to waste IO & CPU. The crc32c is already in the cloud
```
Comment (mcovarr) [inline scripts/reference_disks/create_images.sh:110]: TOL not sure if the original author thought an independent check of the CRC32c was necessary for a `gsutil cp`'d file, but that's certainly not the assumption we've been making recently in DRS-land...

File: scripts/reference_disks/create_images.sh [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+    image_tsv="$1"
+
+    # Set the default zone for future gcloud commands to this machine's zone
+    CLOUDSDK_COMPUTE_ZONE=$(
+        gcloud compute instances list --filter "name=('$(hostname)')" --format "value(zone)"
+    )
+    export CLOUDSDK_COMPUTE_ZONE
+
+    # Get a list of image name prefixes, excluding 'none'
+    image_name_prefixes=$(cut -f 3 "$image_tsv" | grep -v '^none$' | sort | uniq)
```
Comment (mcovarr) [inline scripts/reference_disks/create_images.sh:315]: TOL any reason not to `-u` the `sort`?
```suggestion
    image_name_prefixes=$(cut -f 3 "$image_tsv" | grep -v '^none$' | sort -u)
```

**Review by mcovarr — APPROVED** [review]

File: scripts/reference_disks/create_images.sh (hunk 2 above)
Comment (kshakir) [reply scripts/reference_disks/create_images.sh:315]: Force of habit / muscle memory. I'm often running `sort | uniq -c` to get counts.

File: scripts/reference_disks/create_images.sh (hunk 1 above)
Comment (kshakir) [reply scripts/reference_disks/create_images.sh:110]: No clue about the original intent. The hashes this script generated for the hg19 files match the decimals in fc-dev, and shaved 2h40 minutes off the run on my GCE instance.

**Review by breilly2 — APPROVED** [review]
👍  Looks good to me

**Pull request merged**